slamd/design_assistant/processing/design_assistant_service.py

```diff
@@ -138,13 +144,16 @@ def update_design_assistant_session(cls, value, key=None):
 
         if key == 'design_knowledge':
             DesignAssistantPersistence.update_session_for_design_knowledge_key(value)
+        
+        if key == "formulation":
+            DesignAssistantPersistence.update_session_for_formulation_key(value)
 
     @classmethod
```

Comment:
Die Validierungen für liquids, other usw in der update_design_assistant_session method müssen auch angepasst werden